Library/Formula/librest.rb

```diff
@@ -0,0 +1,40 @@
+class Librest < Formula
+  desc "librest is a library access web services that claim to be RESTful"
```

Comment:
I copied most of the text from the website, but I can live with your suggestion, so I'll change that, too ;)